src/components/panel/panel.spec.js

```diff
@@ -8,37 +8,82 @@ describe('$mdPanel', function() {
   var injectLocals = function($injector) {
     $mdPanel = $injector.get('$mdPanel');
     $rootScope = $injector.get('$rootScope');
+    $rootEl = $injector.get('$rootElement');
   };
 
   beforeEach(function() {
     module('material.components.panel', 'ngSanitize');
 
     inject(injectLocals);
+
+    // By default, the panel is attached to $rootElement, so add it to the DOM.
+    angular.element(document.body).append($rootEl);
   });
 
   it('creates a basic panel', function() {
```

Comment:
I missed this last time, but specs are traditionally of the for `it('should ...', ...`